sdk/cosmos/azure-cosmos-encryption/README.md

```diff
@@ -9,26 +9,180 @@ The encryption plugin is under development and not ready to be consumed yet.
+```
+### CRUD operation on Items
+
+```java
+// Create an item
+Pojo pojo = new Pojo();
+pojo.setSensitiveString("Sensitive Information need to be encrypted");
+cosmosEncryptionAsyncContainer.createItem(pojo)
+.flatMap(response -> {
+        System.out.println("Created item: " + response.getItem());
+        // Read that item 👓
+        return cosmosEncryptionAsyncContainer.readItem(response.getItem().getId(),
+                                  new PartitionKey(response.getItem().getId()),
+                                  Pojo.class);
+    })
+    .flatMap(response -> {
+        System.out.println("Read item: " + response.getItem());
+        // Replace that item 🔁
+        Pojo p = response.getItem();
+        pojo.setSensitiveString("New Sensitive Information");
+        return container.replaceItem(p,
+                                     response.getItem().getId(),
+                                     new PartitionKey(response.getItem().getId()));
+    })
+    // delete that item 💣
+    .flatMap(response -> container.deleteItem(response.getItem().getId(),
+                                              new PartitionKey(response.getItem().getId())))
+    .subscribe();
+```
+
+We have a get started sample app available [here][getting_started_encryption].
+
 
 ## Troubleshooting
-TODO
 
+### General
+
+Azure Cosmos DB is a fast and flexible distributed database that scales seamlessly with guaranteed latency and throughput.
+You do not have to make major architecture changes or write complex code to scale your database with Azure Cosmos DB.
+Scaling up and down is as easy as making a single API call or SDK method call.
+However, because Azure Cosmos DB is accessed via network calls there are client-side optimizations you can make to achieve peak performance when using Azure Cosmos DB Java SDK v4.
+
+- [Performance][perf_guide] guide covers these client-side optimizations.
+
+- [Troubleshooting guide][troubleshooting] covers common issues, workarounds, diagnostic steps, and tools when you use Azure Cosmos DB Java SDK v4 with Azure Cosmos DB SQL API accounts.
+
+### Enable Client Logging
+Azure Cosmos DB Java SDK v4 uses SLF4j as the logging facade that supports logging into popular logging frameworks such as log4j and logback.
+
+For example, if you want to use log4j as the logging framework, add the following libs in your Java classpath.
+
+```xml
+<dependency>
+  <groupId>org.slf4j</groupId>
+  <artifactId>slf4j-log4j12</artifactId>
+  <version>${slf4j.version}</version>
+</dependency>
+<dependency>
+  <groupId>log4j</groupId>
+  <artifactId>log4j</artifactId>
+  <version>${log4j.version}</version>
+</dependency>
+```
+
+Also add a log4j config.
+
+```properties
+# this is a sample log4j configuration
+
+# Set root logger level to INFO and its only appender to A1.
+log4j.rootLogger=INFO, A1
+
+log4j.category.com.azure.cosmos=INFO
```

Comment:
need to add logging for encryption as well here, if we have any logging in encryption project. 
```suggestion
log4j.category.com.azure.cosmos.encryption=INFO
```

Reply:
path log4j.category.com.azure.cosmos will cover all the classes under it including cosmos sdk and cosmos encryption sdk